src/components/Admin/SystemSettings/ChangeSystemSettingsDialog.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextField from '@material-ui/core/TextField';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Button from '@material-ui/core/Button';
+
+const ChangeSystemSettings = props => {
+  const {
+    type,
+    handleChange,
+    handleConfirm,
+    handleClose,
+    keyName,
+    keyValue,
+  } = props;
+  return (
+    <React.Fragment>
+      <DialogTitle>{`${type} key`}</DialogTitle>
+      <DialogContent>
+        <TextField
+          label="Key Name"
+          placeholder="Key Name"
+          margin="normal"
+          value={keyName || ''}
```

Comment:
You should create state variables for this input fields in this component itself. If you pass, keyName from parent, it won't get passed to this, as the function call is only once.

Reply:
@akshatnitd If I localize this state to the inner component, the changes won't be shown in the parent component. How to handle that then?